tools/autograd/derivatives.yaml

```diff
@@ -514,7 +514,11 @@
 - name: squeeze(Tensor self, int64_t dim)
   self: maybe_unsqueeze(grad, dim, self.size(dim) == 1 && self.sizes().size() != 1)
 
-- name: std
+- name: std(Tensor self, bool unbiased)
+  self: var_backward(grad * (result * 2).reciprocal(), self, unbiased)
+
+- name: std(Tensor self, int64_t dim, bool unbiased, bool keepdim)
+  self: var_backward(grad * (destination * 2).reciprocal(), self, dim, unbiased, keepdim)
```

Comment:
is destination the name of the return?  This magic sort of sucks.

Reply:
Yes. Should we just name all the outputs some thing like `output` or `output[0]` for multiple returns?

Reply:
I'm ok with that or just allowing you to manually specify the name for cases like this in the format of native_functions.yaml, e.g.
```
- name: std(...) -> destination
```
or
```
- name: func_with_2_returns(...) -> max, indices
```
I'm not sure if that fits in with how we are doing the code gen, but at least it's not a dangling reference.